walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -53,7 +43,10 @@ public boolean optionalAttendeesOnly(Event event, Collection<String> optionalAtt
     //list of all events
     List<Event> allEvents = new ArrayList<Event>();
     allEvents.addAll(events);
-    //sortByStart(allEvents);
+
+    if (allEvents == null) {
```

Comment:
maybe we should put this before line 32 so that we don't do `allEvents.addAll(events)` if `events` is empty?

```java
if (events.empty()) {
  return answer;
}

Collection<String> reqeustsdAttendeeds = ...
```